personhood/db.go

```
@@ -0,0 +1,67 @@
+package personhood
+
+import (
+	"sync"
+
+	"github.com/dedis/cothority"
+	ol "github.com/dedis/cothority/omniledger/service"
+	"github.com/dedis/onet/log"
+	"github.com/dedis/onet/network"
+	"github.com/dedis/protobuf"
+)
+
+const dbVersion = 1
+
+var storageKey = []byte("storage")
+
+func init() {
+	network.RegisterMessage(&storage1{})
+}
+
+// saves all data.
+func (s *Service) save() error {
+	s.storage.Lock()
+	defer s.storage.Unlock()
+	err := s.Save(storageKey, s.storage)
+	if err != nil {
+		log.Error("Couldn't save data:", err)
```

error is not returned